miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java

```diff
@@ -0,0 +1,260 @@
+      TabbedColumns.MODEL, TabbedColumns.STATUS, TabbedColumns.LAST_SERVICED);
+  private static final Set<String> kitsColumns = Sets.newHashSet(TabbedColumns.NAME, TabbedColumns.VERSION, TabbedColumns.MANUFACTURER,
+      TabbedColumns.PART_NUMBER, TabbedColumns.STOCK_LEVEL, TabbedColumns.PLATFORM);
+  private static final Set<String> indicesColumns = Sets.newHashSet(TabbedColumns.FAMILY, TabbedColumns.NAME, TabbedColumns.SEQUENCE);
+  private static final Set<String> studiesColumns = Sets.newHashSet(Columns.SORT, Columns.NAME, Columns.ALIAS, Columns.DESCRIPTION,
+      Columns.TYPE);
+  private static final Set<String> printersColumns = Sets.newHashSet(Columns.SORT, Columns.PRINTER, Columns.DRIVER, Columns.BACKEND,
+      Columns.AVAILABLE);
+
+  @Before
+  public void setup() {
+    loginAdmin();
+  }
+
+  private ListPage getList(String listTarget) {
+    return ListPage.getListPage(getDriver(), getBaseUrl(), listTarget);
+  }
+
+  private ListTabbedPage getTabbedList(String listTarget) {
+    return ListTabbedPage.getTabbedListPage(getDriver(), getBaseUrl(), listTarget);
+  }
+  @Test
+  public void testListSamplesPageSetup() throws Exception {
+    // Goal: ensure all expected columns are present and no extra
+    testPageSetup("samples", samplesColumns);
+  }
+
+  @Test
+  public void testListSamplesColumnSort() throws Exception {
+    // Goal: ensure all sortable columns can be sorted without errors
+    testColumnsSort("samples");
+  }
+
+  @Test
+  public void testListLibrariesPageSetup() throws Exception {
+    // Goal: ensure all expected columns are present and no extra
+    testPageSetup("libraries", librariesColumns);
+  }
+
+  @Test
+  public void testListLibrariesColumnSort() throws Exception {
+    // Goal: ensure all sortable columns can be sorted without errors
+    testColumnsSort("libraries");
+  }
+
+  @Test
+  public void testListDilutionsPageSetup() throws Exception {
+    // Goal: ensure all expected columns are present and no extra
+    testPageSetup("dilutions", dilutionsColumns);
+  }
+
+  @Test
+  public void testListDilutionsColumnSort() throws Exception {
+    // Goal: ensure all sortable columns can be sorted without errors
+    testColumnsSort("dilutions");
+  }
+
+  @Test
+  public void testListPoolsPageSetup() throws Exception {
+    // Goal: ensure all expected columns are present and no extra
+    testTabbedPageSetup("pools", poolsColumns);
+  }
+
+  @Test
+  public void testListPoolsColumnSort() throws Exception {
+    // Goal: ensure all sortable columns can be sorted without errors
+    testTabbedColumnsSort("pools");
+  }
+
+  @Test
+  public void testListOrdersSetup() throws Exception {
+    // this one is special because the number of order completion states is variable
+    ListTabbedPage page = getTabbedList("poolorders");
+    DataTable table = page.getSelectedTable();
+    List<String> headings = table.getColumnHeadings();
+    // size = order columns + some number of completion state columns
```

Comment:
Could also verify that the non "order columns" are expected and non-duplicate "completion state" columns